src/test/java/MapersTests/MapAbilityBonusTests.java

```diff
@@ -0,0 +1,39 @@
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.example.Character.AbilityBonus;
+import org.example.Character.JSONMapper;
+import org.example.Character.Mappers.MapAbilityBonus;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MapAbilityBonusTests {
+
+    //index 2
+    private final ArrayNode outputMapper = (ArrayNode) JSONMapper.MAPPER.readTree("[{\"ability_score\":{\"index\":\"dex\",\"name\":\"DEX\",\"url\":\"/api/ability-scores/dex\"},\"bonus\":2}]");
+    private final List<AbilityBonus> abilityBonuses = new ArrayList<>();
+    private final MapAbilityBonus testObject = new MapAbilityBonus();
+    private final ArrayNode badOutputMapper = (ArrayNode) JSONMapper.MAPPER.readTree("[{\"ability_score\":{\"index\":\"dex\",\"name\":\"DEX\",\"url\":\"/api/ability-scores/dex\"},\"noslda\":2}]");
+
+    public MapAbilityBonusTests() throws JsonProcessingException {
```

Comment:
Why constructor?

Reply:
Because it throws "JsonMappingException", I can solve this by
- add exception to class default counstructor signature
- surround with try/catch

Reply:
Try catch is better. You won't ever use this constructor

Reply:
ok, corrected!